lib/http/HttpClient_WinInet.cpp

```diff
@@ -32,6 +32,9 @@ class WinInetRequestWrapper
     HINTERNET              m_hWinInetRequest {nullptr};
     SimpleHttpRequest*     m_request;
     BYTE                   m_buffer[1024] {0};
+    DWORD                  m_bufferUsed {0};
+    std::vector<uint8_t>   m_bodyBuffer;
+    bool                   m_readingData {false};
```

Comment:
nit - Can we add a comment after these member variables indicating their purpose. would make it easy to understand the logic. 
PR looks good.